Bad/hidden error message if one has not extracted submodules

The error message is not helpful, and indeet it's not even clear than anything went wrong! The fact that it was an error is hidden in a discrete message on the third to last line, and the actual error is a couple of pages up, out of screen.

Could we change to something that's not cmake (cough, autotools, cough), or failing that, having it fail at the actual error, to make it clear that it failed, and what failed?

```
$ cmake -DCMAKE_INSTALL_PREFIX=$HOME/opt/gnuradio-3.8.1.0 ..                                                     
-- Build type not specified: defaulting to release.                                                                                         
-- Build type set to Release.                                                                                                               
-- Extracting version information from git describe...        
CMake Error at CMakeLists.txt:123 (add_subdirectory):             
  The source directory                                                
                                                                      
    /home/xxx/src/volk/cpu_features         
                                                                                                                                            
  does not contain a CMakeLists.txt file.        
                                                                      
                                                                                                                                            
--                                                                    
-- Python checking for python >= 3.4
-- Python checking for python >= 3.4 - found
--                                                                                                                                          
-- Python checking for mako >= 0.4.2            
-- Python checking for mako >= 0.4.2 - found
-- QA Testing is enabled.                                             
--   Modify using: -DENABLE_TESTING=ON/OFF
-- System profiling is disabled.
--   Modify using: -DENABLE_PROFILING=ON/OFF   
-- Compiler name: GNU
-- Architecture is not x86 or x86_64, Overruled arch 3dnow
-- Architecture is not x86 or x86_64, Overruled arch mmx                                                                                    
-- Architecture is not x86 or x86_64, Overruled arch sse
-- Architecture is not x86 or x86_64, Overruled arch sse2
-- Architecture is not x86 or x86_64, Overruled arch sse3
-- Architecture is not x86 or x86_64, Overruled arch ssse3
-- Architecture is not x86 or x86_64, Overruled arch sse4_a
-- Architecture is not x86 or x86_64, Overruled arch sse4_1
-- Architecture is not x86 or x86_64, Overruled arch sse4_2
-- Architecture is not x86 or x86_64, Overruled arch avx
-- Architecture is not x86 or x86_64, Overruled arch avx512f
-- Architecture is not x86 or x86_64, Overruled arch avx512cd
-- Compiler doesn't support NEON, Overruled arch neon
-- Compiler doesn't support NEON, Overruled arch neonv7
-- Compiler doesn't support NEON, Overruled arch neonv8
-- Available architectures: generic;hardfp;orc;norc
-- Available machines: generic_orc                                    
-- BUILD TYPE = RELEASE
-- Base cflags = -O3 -DNDEBUG  -fcx-limited-range -Wall   
-- BUILD INFO ::: generic_orc ::: GNU ::: -O3 -DNDEBUG  -fcx-limited-range -Wall 
-- Compiler Version: cc (Raspbian 8.3.0-6+rpi1) 8.3.0   
Copyright (C) 2018 Free Software Foundation, Inc.        
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-- c flags:  -fcx-limited-range -Wall;                     
-- asm flags:                                                         
-- c flags:  -fcx-limited-range -Wall;                     
-- asm flags:                                                         
-- Loading version 2.4 into constants...                    
-- Using install prefix: /home/xxx/opt/gnuradio-3.8.1.0   
-- Configuring incomplete, errors occurred!          
See also "/home/xxx/src/volk/build/CMakeFiles/CMakeOutput.log".
See also "/home/xxx/src/volk/build/CMakeFiles/CMakeError.log".
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -122,6 +122,9 @@ else()
   option(VOLK_CPU_FEATURES "Volk uses cpu_features" OFF)
 endif()
 if (VOLK_CPU_FEATURES)
+  if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/cpu_features/CMakeLists.txt" )
+    message(FATAL_ERROR "cpu_features/CMakeLists.txt not found. Did you forget to git clone recursively?\nFix with: git submodule update --init")
+  endif()
   message(STATUS "Building Volk with cpu_features")
   set(BUILD_PIC ON CACHE BOOL
     "Build cpu_features with Position Independent Code (PIC)."

